main.py

```diff
@@ -26,16 +26,23 @@ def create_menu():
     menu.add_option('4', 'Show Read Books', show_read_books)
     menu.add_option('5', 'Show All Books', show_all_books)
     menu.add_option('6', 'Change Book Read Status', change_read)
+    menu.add_option('7', 'Delete Book', delete_book)
     menu.add_option('Q', 'Quit', quit_program)
 
     return menu
 
 
 def add_book():
     new_book = ui.get_book_info()
-    new_book.save()
+    new_book.save()\
+
+def delete_book():
+    try:
+        which_book = ui.get_book_id()
+        BookStore._delete_book(which_book)
```

Comment:
Follow the structure of the other calls. Get the book object and then ask it to delete itself.
```
book = store.get_book_by_id(which_book)
book.delete()
```
In Python, assume methods that start with _ are only used internally in a module, and other modules should not call them, or things may not work as expected.